packages/ia-js-client/src/controllers/item/item.ts

```diff
@@ -1,5 +1,6 @@
 import { MetadataService, Metadata } from '../../services/metadata'
 import { DetailsService, DetailsResponse } from '../../services/details'
+const debug = require('debug')('ia-js-client:item');
```

Comment:
I remember you mentioning that this is an npm module? I can't recall..
Is there a problem with using import statement?

Reply:
I haven't ever seen debug used with import, I think import is async, and I'm pretty sure it has to be loaded before the javascript code (I know the environment variable or localstore has to be set before the "require debug" in order for debug to be enabled. I don't quite understand why but think its to minimise the impact on production code in the case that debugging is off.